assets/react/v3/entries/course-builder/components/additional/MeetingForm.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { RefObject, useEffect, useRef } from 'react';
+import { css } from '@emotion/react';
+import { __ } from '@wordpress/i18n';
+import { Controller } from 'react-hook-form';
+
+import Button from '@Atoms/Button';
+
+import FormCheckbox from '@Components/fields/FormCheckbox';
+import FormDateInput from '@Components/fields/FormDateInput';
+import FormInput from '@Components/fields/FormInput';
+import FormTextareaInput from '@Components/fields/FormTextareaInput';
+import FormTimeInput from '@Components/fields/FormTimeInput';
+
+import { borderRadius, colorPalate, colorTokens, fontSize, shadow, spacing } from '@Config/styles';
+import { typography } from '@Config/typography';
+import Show from '@Controls/Show';
+
+import { useFormWithGlobalError } from '@Hooks/useFormWithGlobalError';
+
+import { Meeting } from './LiveClass';
+import { styleUtils } from '@Utils/style-utils';
+import FormSelectInput from '@Components/fields/FormSelectInput';
+
+export type MeetingType = 'zoom' | 'google_meet' | 'jitsi';
+
+interface MeetingFormFieldProps {
+  meeting_name: string;
+  meeting_summary: string;
+  meeting_date: string;
+  meeting_time_from: string;
+  meeting_time_to: string;
+  meeting_enrolledAsAttendee: boolean;
+  meeting_autoRecording: string;
+  meeting_password: string;
+  meeting_host: string;
```

Comment:
Use camelCase naming

Reply:
These are used as Form Fields Props. In all other places, form field data properties are named with snake cases.